### PR TITLE
Sass 2941

### PR DESCRIPTION
**Pull request by satheeshgottemukkula**
### Description
Pensions - payments into overseas pensions gateway page

[SASS-2941]

##### Before Reviewing
- [ ]  Have you pulled the branch down?
- [ ]  Have you assigned yourself to the PR?
- [ ]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [ ]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- []  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -118,10 +120,18 @@ pensions.pensionSummary.incomeFromPensions = Income from pensions
 pensions.pensionSummary.pensionAnnualAllowance = Pension annual allowance
 pensions.pensionSummary.pensionLifetimeAllowance = Pension lifetime allowance
 pensions.pensionSummary.unauthorisedPaymentsFromPensions = Unauthorised payments from pensions
-pensions.pensionSummary.paymentsToOverseasPensions = Payments into overseas pensions
+pensions.pensionSummary.paymentsToOverseasPensions = Taliadau i bensiynau tramor
 
 
 ########    PAYMENTS INTO PENSIONS    ########
+pensions.paymentsIntoOverseasPensions.statusPage.error.agent = Dewiswch ‘Iawn’ os gwnaeth eich cleient neu ei gyflogwr dalu i mewn i gynllun pensiwn tramor
+pensions.paymentsIntoOverseasPensions.statusPage.error.individual = Dewiswch ‘Iawn’ os gwnaethoch chi neu''ch cyflogwr dalu i mewn i gynllun pensiwn tramor
```
Comment (jhkldev) [inline conf/messages.cy:128]: The word `statusPage` could be removed to match the other properties. So each property would start with `pensions.paymentsIntoOverseasPensions.`

**Review by jhkldev — CHANGES_REQUESTED** [review]

File: app/forms/PaymentsIntoOverseasPensionsFormProvider.scala [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import play.api.data.Form
+
+import javax.inject.Singleton
+
+@Singleton
+class PaymentsIntoOverseasPensionsFormProvider {
+
+  def paymentsIntoOverseasPensionsForm(isAgent: Boolean): Form[Boolean] = YesNoForm.yesNoForm(
+    missingInputError = s"pensions.paymentsIntoOverseasPensions.statusPage.error.${if (isAgent) "agent" else "individual"}"
```
Comment (jhkldev) [inline app/forms/PaymentsIntoOverseasPensionsFormProvider.scala:27]: Needs updating when `statusPage` is removed.

**Review by jhkldev — CHANGES_REQUESTED** [review]

File: conf/messages.cy (hunk 1 above)
Comment (satheeshgottemukkula) [reply conf/messages.cy:128]: Agree, will remove

File: app/forms/PaymentsIntoOverseasPensionsFormProvider.scala (hunk 2 above)
Comment (satheeshgottemukkula) [reply app/forms/PaymentsIntoOverseasPensionsFormProvider.scala:27]: Agree and will remove

**Review by jhkldev — DISMISSED** [review]

**Review by jhkldev — APPROVED** [review]

**Pull request merged**